index.js

```diff
@@ -146,6 +129,27 @@ function processFile (file, config, watcher) {
   })
 }
 
+function config () {
```

Comment:
👍  If you check for `postcss.config.js` first, `-p|-m|-s|-t` (options in conditional) could be used as context if set and config is present, but can be done later/should be non breaking